image_provisioner/playbooks/roles/repo-install/tasks/main.yaml

```diff
@@ -26,6 +26,15 @@
     mode: 0644
   when: internal_image | bool
 
+- name: install rhel-7-update-kernel-repo
+  template:
+    src: etc/yum.repos.d/rhel-7-update-kernel.repo.j2
+    dest: /etc/yum.repos.d/rhel-7-update-kernel.repo
+    owner: root
```

Comment:
`root:root` is typically the default owner of everything so this is not necessary but doesn't hurt